tools/mtmd/coreml/mtmd_coreml.mm

```diff
@@ -0,0 +1,98 @@
+#import <CoreML/CoreML.h>
+#import <Accelerate/Accelerate.h>
+#import "mtmd_coreml.h"
+#import "coreml_minicpmv40_vit_f16.h"
+#include <stdlib.h>
+
+#if __cplusplus
+extern "C" {
```

Comment:
I'm not quite sure if this `extern "C"` is needed. if it is not intended to be used outside of the library (i.e. exposed as a public API), I prefer to replace this with a `namespace mtmd_corelm`